playwright/tests/page-objects/toolbar.ts

```diff
@@ -34,7 +34,7 @@ export class Toolbar {
     await this.toolbar.getByRole('button', { name: 'Device mode' }).click();
   }
 
-  async openPalette(name: string) {
+  async openPalette(name: 'All Components' | 'Structures' | 'Elements' | 'Actions' | 'Composites') {
```

Comment:
I think 'All Components' a bulky term that blows up the palette. IMHO 'All' would be better ... but I think this is not the time to discuss it :wink:

Reply:
Would be fine for me. `All Components` is streamlined with the process editor where it is called `All Elements`.
I'm not sure about `Elements`, but we can change this later too...

Reply:
what do you think @ivy-bb ; shouldn't we abbreviate this bulky term? "All Components" ?

![toolbar](https://private-user-images.githubusercontent.com/42733123/384885980-2fd80431-9309-4ec0-a57d-a56f738d6dc5.png)

Reply:
IMO All Components is fine. Like All Elements